src/js/main.js

```diff
@@ -2,47 +2,32 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function($, Mozilla, Clipboard, dataLayer) {
+
+if (typeof window.dataLayer === 'undefined') {
+    window.dataLayer = [];
+}
+
+(function($, Mozilla, Clipboard, dataLayer, Mzp) {
     'use strict';
 
-    var $episodeLinks = $('.episode-links');
-    var $subscribeLinks = $('.subscribe-links');
-    var $socialShareLinks = $('.social-share-links');
+    var $subscribeButtons = $('.episode-subscribe');
+    var $showNotesLink = $('.episode-shownotes-link');
+    var $subscribeLinks = $('[data-link-type="subscribe"]');
+    var $shareLinks = $('[data-link-type="social share"]');
 
     // set up clipboard.js
-    var copylinkClipboard = new Clipboard('.copylink');
+    var copylinkClipboard = new Clipboard('.js-action-copy a');
 
     // display related copy success message when clicking copy icon
     copylinkClipboard.on('success', function() {
```

Comment:
This...isn't working. I'm not exactly sure what the issue is, but it might be related to the `e.preventDefault()` call below (line 49).